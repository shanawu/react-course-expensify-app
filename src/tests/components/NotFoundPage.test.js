import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import NotFoundPage from "../../components/NotFoundPage";

test("should render Not Found page", () => {
  const { container } = render(
    <BrowserRouter basename="/">
      <NotFoundPage />
    </BrowserRouter>
  );

  expect(container).toMatchSnapshot();
});
