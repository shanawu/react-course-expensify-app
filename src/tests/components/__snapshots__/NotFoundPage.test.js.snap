// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found page 1`] = `
<div>
  <div>
    404 - 
    <a
      href="/"
    >
      Go home
    </a>
  </div>
</div>
`;
